error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/MockitoVerifyNoInteractionsUsageTest.java

```diff
@@ -0,0 +1,193 @@
+  private final BugCheckerRefactoringTestHelper refactoringTestHelper =
+      BugCheckerRefactoringTestHelper.newInstance(
+          MockitoVerifyNoInteractionsUsage.class, getClass());
+
+  @Test
+  void identification() {
+    CompilationTestHelper.newInstance(MockitoVerifyNoInteractionsUsage.class, getClass())
+        .addSourceLines(
+            "A.java",
+            "import static org.mockito.Mockito.mock;",
+            "import static org.mockito.Mockito.verifyNoInteractions;",
+            "",
+            "import org.junit.jupiter.api.Test;",
+            "",
+            "class A {",
+            "  @Test",
+            "  void a() {}",
+            "  @Test",
+            "  void b() {",
+            "    Runnable runnable = mock(Runnable.class);",
+            "    verifyNoInteractions(runnable);",
+            "  }",
+            "  @Test",
+            "  void c() {",
+            "    Runnable runnable = mock(Runnable.class);",
+            "    Runnable runnable1 = mock(Runnable.class);",
+            "    verifyNoInteractions(runnable, runnable1);",
+            "  }",
+            "  @Test",
+            "  // BUG: Diagnostic contains:",
+            "  void d() {",
+            "    Runnable runnable1 = mock(Runnable.class);",
+            "    Runnable runnable2 = mock(Runnable.class);",
+            "    verifyNoInteractions(runnable1);",
+            "    verifyNoInteractions(runnable2);",
+            "  }",
+            "  @Test",
+            "  // BUG: Diagnostic contains:",
+            "  void e() {",
+            "    Runnable runnable1 = mock(Runnable.class);",
+            "    verifyNoInteractions(runnable1);",
+            "    Runnable runnable2 = mock(Runnable.class);",
+            "    verifyNoInteractions(runnable2);",
+            "  }",
+            "  @Test",
+            "  // BUG: Diagnostic contains:",
+            "  void f() {",
+            "    Runnable runnable1 = mock(Runnable.class);",
+            "    verifyNoInteractions(runnable1);",
+            "    Runnable runnable2 = mock(Runnable.class);",
+            "    Runnable runnable3 = mock(Runnable.class);",
+            "    verifyNoInteractions(runnable2, runnable3);",
+            "    Runnable runnable4 = mock(Runnable.class);",
+            "    Runnable runnable5 = mock(Runnable.class);",
+            "    verifyNoInteractions(runnable4);",
+            "    verifyNoInteractions(runnable5);",
+            "  }",
+            "}")
+        .doTest();
+  }
+
+  @Test
+  void replaceSequentialCalls() {
+    refactoringTestHelper
+        .addInputLines(
+            "A.java",
+            "import static org.mockito.Mockito.mock;",
+            "import static org.mockito.Mockito.verifyNoInteractions;",
+            "",
+            "import org.junit.jupiter.api.Test;",
+            "",
+            "class A {",
+            "  @Test",
+            "  void m() {",
+            "    Runnable runnable1 = mock(Runnable.class);",
```

Comment:
Nice tests @Kamil-Gabaydullin ! 
 
You already added quite some cases which is awesome.
What could even further improve the tests is by having other test statements between all these mocks. 
In tests there are also other methods that are mocked and used, I'm curious how those cases would be handled in replacement tests.

So in other words, now it contains actual mock statements and the `verifyNoInteractions` invocations. It would be nice if we add some `when(...)` calls and a random `assertThat` call maybe. That would make it a little more realistic tests and tests also the non-happy path a bit more.